src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryCachingConfiguration.java

```diff
@@ -54,6 +70,17 @@ public Boolean isRefreshEnabled() {
     public String getExcludedVersionsStr() {
         return excludedVersionsStr;
     }
+    public String getIncludedVersionsStr() {
```

Comment:
do we want a `@CheckForNull` here?

Reply:
There isn't any such annotations on the other one?
It I add one here I feel like I would need to add it everywhere :/

Reply:
Fair point, you can disregard my comment :)